### ISSUE TITLE
Collection Load order jumbled.

### ISSUE THREAD

**Issue by Ryuktheshinigamix** [Hypocrita20XX/Conexus#31]
I cant find any problems in the log or the collection i have set but when i load up the game and check the mod load order to activate them the whole order is jumbled up. All the Folders in the mod folder seem correctly in order just as the collection so i dont know what went wrong. I have checked things and re-run Conexus 3 times.

**Comment by Dragnaruz** [comment]
did you find a fix for it? i have the same problem, mine is following a alphabetical order instead.

**Comment by Ryuktheshinigamix** [comment]
> did you find a fix for it? i have the same problem, mine is following a alphabetical order instead.

Yes I did. Its not easy but its better then nothing. [Solution](https://steamcommunity.com/sharedfiles/filedetails/?id=1982294244)
